src/build.js

```diff
@@ -43,6 +45,74 @@ export async function generateBuildMap(configObj) {
   return await iterateDir(configObj.inputDir);
 }
 
+export async function generateBundleOutputObj(buildMap) {
```

Comment:
Can you document what `buildMap` is or pick a more expressive name? E.g. `transformedContentByPath` (?)

Reply:
I've refactored using a better name and more comments throughout